.gitignore

```diff
@@ -31,3 +31,7 @@ docs/_site/
 
 #Logging
 logs
+
+#Roo long term memory bank
+memory-bank/
+.clinerules-*
```

Comment:
Do you mind leaving this part out of the PR? Eventually we may want to check in some of the .clinerules. Not sure about the memory bank, but probably deserving of a separate discussion.